client/my-sites/plans-v2/plans-filter-bar/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+
+import React, { useState } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SegmentedControl from 'components/segmented-control';
+import SelectDropdown from 'components/select-dropdown';
+import { TERM_MONTHLY, TERM_ANNUALLY } from 'lib/plans/constants';
+import type { Duration, ProductType } from '../types';
```

Comment:
It's my personal taste- I like having a separate "Type dependencies" area to clearly distinguish types vs actual imports.

Reply:
I like that too.